packages/core/admin/server/src/controllers/admin.ts

```diff
@@ -154,12 +154,26 @@ export default {
   async plugins(ctx: Context) {
     const enabledPlugins = strapi.config.get('enabledPlugins') as any;
 
-    const plugins = Object.entries(enabledPlugins).map(([key, plugin]: any) => ({
-      name: plugin.info.name || key,
-      displayName: plugin.info.displayName || plugin.info.name || key,
-      description: plugin.info.description || '',
-      packageName: plugin.info.packageName,
-    }));
+    // List of core plugins that are always enabled,
+    // and so it's not necessary to display them in the plugins list
+    const CORE_PLUGINS = [
```

Comment:
This list will need to be maintained with the same list as https://github.com/strapi/strapi/blob/v5/main/packages/core/core/src/loaders/plugins/get-enabled-plugins.ts shall we write that?

Reply:
good point, will add a comment